src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -333,11 +352,18 @@ public WorkspacePanel(final MCreator mcreator) {
 			}
 		});
 
+		renameFolder.addActionListener(e -> {
+		    if (list.getSelectedValue() instanceof FolderElement) {

```

Comment:
Call this above, store IElement as local var and then pass it as casted FolderElement so you don't need to cast it again later

Reply:
idk if i did it well, but yeah ok